plugins/cilium-cni/04-flannel-cilium-cni.conflist

```diff
@@ -0,0 +1,23 @@
+{
+  "cniVersion": "0.3.1",
+  "name": "flannel-cilium",
+  "plugins": [
+    {
+      "type": "flannel",
+      "delegate": {
+         "hairpinMode": true,
```

Comment:
This may be because I am not an expert on this stuff, but please document why these options are needed for code-maintenance purposes.